ras_party/controllers/controller.py

```diff
@@ -569,6 +570,7 @@ def _send_email_verification(party_uuid, email):
     """
 
     verification_url = _create_verification_url(email)
+    # verification_url = "http://this-is-a-test"
```

Comment:
I can't see any value in keeping this comment

Reply:
removed